src/compiler/parser.cc

```diff
@@ -981,26 +981,32 @@ Declaration* Parser::parse_declaration(bool is_abstract) {
 }
 
 Class* Parser::parse_class_interface_monitor_or_mixin(bool is_abstract) {
+  auto at = [&](Symbol symbol) -> bool {
```

Comment:
I'd probably add a helper on Parser for this instead.

Reply:
Added `at_pseudo` (since this is basically a look for a pseudo keyword.